### PR TITLE
Added user management

### LINKED ISSUE #308: Add the list of the user's roles to user page
As in permissions page.

### LINKED ISSUE #310: Merge admin's and user's user data page
There are only a few differences between them (some modifiable fields and info messages)

### PR DESCRIPTION
**Pull request by luksan47**
Fixes #308 
Fixes #310 

Also reviewed and modified some permissions.

### CODE REVIEW

File: resources/views/admin/user/list.blade.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                            </td>
+                            <td>
+                                <div class="right">
+                                    @can('view', $user)
+                                    <a href="{{ route('admin.user.show', ['id' => $user->id]) }}" class="btn-floating waves-effect">
+                                        <i class="material-icons">remove_red_eye</i></a>
+                                    @endcan
+                                    @can('viewPermissionFor', $user)
+                                    <a href="{{ route('admin.permissions.show', ['id' => $user->id]) }}" class="btn-floating red waves-effect">
+                                        <i class="material-icons">lock</i></a>
```
Comment (kdmnk) [inline resources/views/admin/user/list.blade.php:31]: This icon's meaning is not clear here. Maybe we can use regular buttons, or delete this from here: we can access the permissions through the user page.

File: resources/views/user/roles.blade.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        @endif
+    </span>
+    @endforeach
+</td>
+<td>
+    <a href="{{ route('admin.permissions.show', $user->id) }}"
+        class="btn-floating waves-effect waves-light right" @cannot('viewPermissionFor', $user) disabled @endcan>
+        <i class="material-icons">edit</i>
+    </a>
+</td>
```
Comment (kdmnk) [inline resources/views/user/roles.blade.php:16]: I think we should add this to the user's page so everyone could see their roles.

File: resources/views/user/personal-information.blade.php [hunk 3]
```diff
@@ -0,0 +1,110 @@
+{{-- Personal information --}}
+@can('viewPersonalInformation', $user)
+<div class="card">
+    <div class="card-content">
+        <div class="card-title">@lang('user.personal_information')</div>
```
Comment (kdmnk) [inline resources/views/user/personal-information.blade.php:5]: This appears in English in Hungarian locale.

**Review by kdmnk — APPROVED** [review]
Nice!
Left some comments. And one more thing: Could you add some placeholder '-' or 'no data' text for empty values? Thanks

File: resources/views/admin/user/list.blade.php (hunk 1 above)
Comment (luksan47) [reply resources/views/admin/user/list.blade.php:31]: Okay, I'll just remove.

File: resources/views/user/personal-information.blade.php (hunk 3 above)
Comment (luksan47) [reply resources/views/user/personal-information.blade.php:5]: Yes, I noticed, the whole user locale contains english translations. This should be fixed, but I left it for some other issue, will open one.

**Pull request merged**